### PR TITLE
Fix OpenBSD

### PR DESCRIPTION
**Pull request by vazub**
- Makefile gets OpenBSD-specific embedding
- Remove bash reference to make Fin more portable and shell-agnostic

### CODE REVIEW

**Review by andydotxyz — APPROVED** [review]
This is a great improvement thanks.
Before we merge I wanted to check - is it not possible to have the Makefile do the right thing in `embed` based on the OS? i.e. have some conditional code so OpenBSD users don't have to call different tasks?

**Comment by vazub** [comment]
Unfortunately, this is not as straightforward as it may seem. GNU Make and BSD Make have different syntax for conditionals, and trying to make it work for both without a hassle is an exercise in futility, for arguably very little gain. So it would either require of OpenBSD user to install additional `gmake` dependency and use it, or going with my less elegant, but quite isolated approach. Moreover, OpenBSD setup is a bit more complicated - requiring a separate build first, then setuid, and then moving it to a filesystem that doesn't have `nosuid` flag - all that to make auth work. So a simple `go run` is not viable in this case. 

To summarize: I don't have a more elegant solution for the time being, but if I find one, I will definitely change this moving forward.
